content/_includes/site/ccs.njk

```diff
@@ -4,7 +4,8 @@
   <summary data-btn="menu xsmall summary">
     Light/Dark
   </summary>
-  <form data-dropdown="pad rtl">
+  <fieldset data-dropdown="pad rtl">
+    <legend>Color Settings</legend>
```

Comment:
@oluoluoxenfree Since the `forms.fieldset` macro already adds a `<fieldset>`, this will result in nested fieldsets. Maybe we just want a `<div>` here instead?

cc @mirisuzanne

Reply:
I can try that!